server/repos/bots.js

```diff
@@ -18,7 +18,7 @@ function create(userId, params) {
 }
 
 async function update(userId, _id, params) {
-  await Bot.updateOne({ _id, user: userId }, params);
+  await Bot.updateOne({ _id, user: userId }, { ...params, updatedAt: new Date() });
```

Comment:
this doesn't work automatically?

Reply:
@juanprq no, the timestamps get updated for `save`, `update` and `findOneAndUpdate`, but not for `updateOne`. For that reason I set it manually. An alternative would be to use `update` instead of `updateOne`.